.storybook/config.js

```diff
@@ -1,5 +1,6 @@
 import { configure } from '@storybook/react';
 
+window.jest = { fn: () => {} };
```

Comment:
are you mocking all window objects?? did you mean to do this instead: https://github.com/theforeman/foreman/pull/5019/files#diff-a7e25ac69ff4b688eb41c907225d770bR19